README.md

```diff
@@ -81,8 +81,9 @@ pnpm --filter open-api test
 # run all tests
 pnpm -r --workspace-concurrency=1 test
 
-# pull in latest openapi specs for auth server:
+# pull in latest openapi specs:
 pnpm --filter auth fetch-schemas
+pnpm --filter backend fetch-schemas
```

Comment:
can we add a root level command?
`"localenv:fetch-op-schemas": "pnpm --filter auth --filter backend fetch-schemas"` , and then include it as part of the local env setup readme?


https://interledgerfoundation.slack.com/archives/C0302LMKBT7/p1670414356297499?thread_ts=1670407497.730339&cid=C0302LMKBT7